Test failures on Python 3.11

```
________________ CallbackHandlerTest.test_callback_id_property _________________
AttributeError: property 'callback_id' of 'CallbackHandler' object has no setter

During handling of the above exception, another exception occurred:

self = <tests.mobly.controllers.android_device_lib.callback_handler_test.CallbackHandlerTest testMethod=test_callback_id_property>

    def test_callback_id_property(self):
      mock_event_client = mock.Mock()
      handler = callback_handler.CallbackHandler(callback_id=MOCK_CALLBACK_ID,
                                                 event_client=mock_event_client,
                                                 ret_value=None,
                                                 method_name=None,
                                                 ad=mock.Mock())
      self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
>     with self.assertRaisesRegex(AttributeError, "can't set attribute"):
E     AssertionError: "can't set attribute" does not match "property 'callback_id' of 'CallbackHandler' object has no setter"

tests/mobly/controllers/android_device_lib/callback_handler_test.py:50: AssertionError
______________ CallbackHandlerBaseTest.test_callback_id_property _______________
AttributeError: property 'callback_id' of 'FakeCallbackHandler' object has no setter

During handling of the above exception, another exception occurred:

self = <tests.mobly.snippet.callback_handler_base_test.CallbackHandlerBaseTest testMethod=test_callback_id_property>

    def test_callback_id_property(self):
      handler = FakeCallbackHandler(callback_id=MOCK_CALLBACK_ID)
      self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
>     with self.assertRaisesRegex(AttributeError, "can't set attribute"):
E     AssertionError: "can't set attribute" does not match "property 'callback_id' of 'FakeCallbackHandler' object has no setter"

tests/mobly/snippet/callback_handler_base_test.py:89: AssertionError
________________ CallbackHandlerBaseTest.test_timeout_property _________________
AttributeError: property 'rpc_max_timeout_sec' of 'FakeCallbackHandler' object has no setter

During handling of the above exception, another exception occurred:

self = <tests.mobly.snippet.callback_handler_base_test.CallbackHandlerBaseTest testMethod=test_timeout_property>

    def test_timeout_property(self):
      handler = FakeCallbackHandler(rpc_max_timeout_sec=20,
                                    default_timeout_sec=10)
      self.assertEqual(handler.rpc_max_timeout_sec, 20)
      self.assertEqual(handler.default_timeout_sec, 10)
>     with self.assertRaisesRegex(AttributeError, "can't set attribute"):
E     AssertionError: "can't set attribute" does not match "property 'rpc_max_timeout_sec' of 'FakeCallbackHandler' object has no setter"

tests/mobly/snippet/callback_handler_base_test.py:80: AssertionError
```

--- a/tests/mobly/controllers/android_device_lib/callback_handler_test.py
+++ b/tests/mobly/controllers/android_device_lib/callback_handler_test.py
@@ -47,7 +47,7 @@ class CallbackHandlerTest(unittest.TestCase):
                                                method_name=None,
                                                ad=mock.Mock())
     self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
-    with self.assertRaisesRegex(AttributeError, "can't set attribute"):
+    with self.assertRaisesRegex(AttributeError, "callback_id"):
       handler.callback_id = 'ha'
 
   def test_event_dict_to_snippet_event(self):

--- a/tests/mobly/controllers/android_device_lib/callback_handler_test.py
+++ b/tests/mobly/controllers/android_device_lib/callback_handler_test.py
@@ -47,7 +47,7 @@ class CallbackHandlerTest(unittest.TestCase):
                                                method_name=None,
                                                ad=mock.Mock())
     self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
-    with self.assertRaisesRegex(AttributeError, "callback_id"):
+    with self.assertRaises(AttributeError):
       handler.callback_id = 'ha'
 
   def test_event_dict_to_snippet_event(self):

--- a/tests/mobly/snippet/callback_handler_base_test.py
+++ b/tests/mobly/snippet/callback_handler_base_test.py
@@ -77,16 +77,16 @@ class CallbackHandlerBaseTest(unittest.TestCase):
                                   default_timeout_sec=10)
     self.assertEqual(handler.rpc_max_timeout_sec, 20)
     self.assertEqual(handler.default_timeout_sec, 10)
-    with self.assertRaisesRegex(AttributeError, "can't set attribute"):
+    with self.assertRaisesRegex(AttributeError, "rpc_max_timeout_sec"):
       handler.rpc_max_timeout_sec = 5
 
-    with self.assertRaisesRegex(AttributeError, "can't set attribute"):
+    with self.assertRaisesRegex(AttributeError, "default_timeout_sec"):
       handler.default_timeout_sec = 5
 
   def test_callback_id_property(self):
     handler = FakeCallbackHandler(callback_id=MOCK_CALLBACK_ID)
     self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
-    with self.assertRaisesRegex(AttributeError, "can't set attribute"):
+    with self.assertRaisesRegex(AttributeError, "callback_id"):
       handler.callback_id = 'ha'
 
   def test_event_dict_to_snippet_event(self):

--- a/tests/mobly/snippet/callback_handler_base_test.py
+++ b/tests/mobly/snippet/callback_handler_base_test.py
@@ -77,16 +77,16 @@ class CallbackHandlerBaseTest(unittest.TestCase):
                                   default_timeout_sec=10)
     self.assertEqual(handler.rpc_max_timeout_sec, 20)
     self.assertEqual(handler.default_timeout_sec, 10)
-    with self.assertRaisesRegex(AttributeError, "rpc_max_timeout_sec"):
+    with self.assertRaises(AttributeError):
       handler.rpc_max_timeout_sec = 5
 
-    with self.assertRaisesRegex(AttributeError, "default_timeout_sec"):
+    with self.assertRaises(AttributeError):
       handler.default_timeout_sec = 5
 
   def test_callback_id_property(self):
     handler = FakeCallbackHandler(callback_id=MOCK_CALLBACK_ID)
     self.assertEqual(handler.callback_id, MOCK_CALLBACK_ID)
-    with self.assertRaisesRegex(AttributeError, "callback_id"):
+    with self.assertRaises(AttributeError):
       handler.callback_id = 'ha'
 
   def test_event_dict_to_snippet_event(self):

